### PR TITLE
add ability to configure deployment strategy

### PR DESCRIPTION
**Pull request by varditn**
Signed-off-by: Vardit Natanson <v.natanson@travelaudience.com>

#### What this PR does / why we need it:
* Adding ability to adjust deployment strategy
* Adding option to state pvc namespace in added pv template.
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/sonatype-nexus/README.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ The following table lists the configurable parameters of the Nexus chart and the
 | Parameter                                   | Description                         | Default                                 |
 | ------------------------------------------  | ----------------------------------  | ----------------------------------------|
 | `replicaCount`                              | Number of Nexus service replicas    | `1`                                     |
+| `deploymentStrategy`                        | Deployment Strategy     |  `rollingUpdate` |
+| `namespace`                        | pvc namespace     |  `default` |
```
Comment (rjkernick) [inline stable/sonatype-nexus/README.md]: Not sure I understand the use case of manually setting the pvc namespace. Helm charts generally control resource within the same namespace.

File: stable/sonatype-nexus/README.md [hunk 2]
```diff
@@ -63,6 +63,8 @@ The following table lists the configurable parameters of the Nexus chart and the
 | Parameter                                   | Description                         | Default                                 |
 | ------------------------------------------  | ----------------------------------  | ----------------------------------------|
 | `replicaCount`                              | Number of Nexus service replicas    | `1`                                     |
+| `deploymentStrategy`                        | Deployment Strategy     |  `rollingUpdate` |
```
Comment (rjkernick) [inline stable/sonatype-nexus/README.md:66]: The deployment strategy is a nice addition.

File: stable/sonatype-nexus/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: sonatype-nexus
-version: 1.9.0
+version: 1.9.1
```
Comment (rjkernick) [inline stable/sonatype-nexus/Chart.yaml]: Please bump the minor version as this adds a feature

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by rjkernick** [comment]
I like the deployment strategy addition. However, the pvc namespace seems like a change that would break the ability to easily update from an existing version.

**Comment by TAvardit** [comment]
> I like the deployment strategy addition. However, the pvc namespace seems like a change that would break the ability to easily update from an existing version.

You are totally right,  I changed it to use Helm pre-defined values instead.

**Review by rjkernick — APPROVED** [review]

**Pull request merged**